src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,838 @@
- :settings-page/git-switcher-label "Habilitar auto-consolidação no Git"
- :settings-page/git-commit-delay "Intervalo em segundos para auto-consolidação"
- :settings-page/show-brackets "Mostrar parênteses retos"
+ :content/click-to-edit "Clique para editar"
+ :context-menu/make-a-flashcard "Criar um Flashcard"
+ :context-menu/toggle-number-list "Alternar lista numerada"
+ :context-menu/preview-flashcard "Visualizar Flashcard"
+ :context-menu/make-a-template "Criar um Modelo"
+ :context-menu/input-template-name "Qual é o nome do modelo?"
+ :context-menu/template-include-parent-block "Incluir o bloco pai no modelo?"
+ :context-menu/template-exists-warning "Modelo já existe!"
+ :settings-page/git-tip "Se você tiver a Sincronização do Logseq ativada, você pode visualizar o histórico de edições de uma página diretamente. Esta seção é apenas para usuários técnicos."
+ :settings-page/git-desc-1 "Para visualizar o histórico de edições de uma página, clique nos três pontos horizontais no canto superior direito e selecione \"Ver histórico da página\"."
+ :settings-page/git-desc-2 "Para usuários avançados, o Logseq também oferece suporte para uso do "
+ :settings-page/git-desc-3 " para controle de versão. Use o Git por sua própria conta e risco, uma vez que questões gerais do Git não são suportadas pela equipe do Logseq."
+ :settings-page/git-switcher-label "Ativar confirmação automática do Git"
+ :settings-page/git-commit-delay "Segundos para confirmação automática do Git"
+ :settings-page/git-confirm "Você precisa reiniciar o aplicativo após atualizar as configurações do Git."
+  :settings-page/edit-config-edn "Editar config.edn"
+ :settings-page/edit-global-config-edn "Editar config.edn global"
+ :settings-page/edit-custom-css "Editar custom.css"
+ :settings-page/edit-export-css "Editar export.css"
+ :settings-page/edit-setting "Editar"
+ :settings-page/custom-configuration "Configuração personalizada"
+ :settings-page/custom-global-configuration "Configuração global personalizada"
+ :settings-page/theme-light "Claro"
+ :settings-page/theme-dark "Escuro"
+ :settings-page/theme-system "Sistema"
+ :settings-page/custom-theme "Tema personalizado"
+ :settings-page/export-theme "Exportar tema"
+ :settings-page/show-brackets "Mostrar colchetes"
  :settings-page/spell-checker "Verificador ortográfico"
- :settings-page/disable-sentry "Enviar dados de utilização e diagnósticos para Logseq"
- :settings-page/preferred-outdenting "Ativar dedentação lógica"
- :settings-page/auto-expand-block-refs "Expandir as referências de bloco automaticamente ao aumentar o zoom"
+ :settings-page/auto-updater "Atualização automática"
+ :settings-page/disable-sentry "Enviar dados de uso e diagnósticos para o Logseq"
+ :settings-page/disable-sentry-desc "O Logseq nunca coletará o banco de dados do seu gráfico local nem venderá seus dados."
+ :settings-page/preferred-outdenting "Rebaixamento lógico"
```

Comment:
Perhaps a better translation would be "Recuar indentação".

Reply:
I reviewed it here and readjusted it accordingly.